### PR TITLE
[SMALLFIX] Clean up common test utils test

### PR DESCRIPTION
**Pull request by aaudiber**
Break up monolithic test into its parts

### CODE REVIEW

File: core/common/src/test/java/alluxio/CommonTestUtils.java [hunk 1]
```diff
@@ -121,7 +122,11 @@ public static void waitFor(String description, Function<Void, Boolean> condition
       }
       field.setAccessible(true);
       T instance = createBaseObject(clazz);
-      field.set(instance, getValuesForFieldType(field.getType()).get(1));
+      try {
```
Comment (yupeng9) [inline core/common/src/test/java/alluxio/CommonTestUtils.java]: Any reason not to throw the exceptions?

Comment (aaudiber) [inline core/common/src/test/java/alluxio/CommonTestUtils.java]: The exception is still thrown, but now it isn't checked. Throwing a checked exception here places an unnecessary burden on callers, which never care about the specific type of exception thrown.

**Comment by apc999** [comment]
+1 LGTM

File: core/common/src/test/java/alluxio/CommonTestUtils.java (hunk 1 above)
Comment (yupeng9) [inline core/common/src/test/java/alluxio/CommonTestUtils.java]: sgtm

**Pull request merged**